Apps/WebClient/src/ClientApp/app/components/timeline/medication.vue

```diff
@@ -235,6 +259,24 @@ export default class MedicationTimelineComponent extends Vue {
     return faPills;
   }
 
+  private toggleComments(): void {
+    this.commentsVisible = !this.commentsVisible;
+    this.hasErrors = false;
+    if (!this.commentsVisible) {
+      return;
+    }
+  }
+
+  private sortComments() {
```

Comment:
these lines could also go into the comment**s** component

Reply:
btw, you can ignore this one as it is paired to making the comment list a component on its own